pages/api/discord/user.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { redis } from "../../../libs/redis";
+
+const DISCORD_API = "https://discord.com/api/v10";
+
+const getTokenByAddress = async (address: string) => {
+  const redisValue = await redis.get(address);
```

Comment:
Do we need to enable redis on Vercel somewhere?

Reply:
I am using an instance of [Upstash](https://upstash.com)